contrib/sideload_csv2json.py

```diff
@@ -0,0 +1,92 @@
+from copy import deepcopy
+import csv
+import json
+import ast
+
+startrow = 0
+endrow = 1000000000
+filename = "Starling-Caption-Credit.csv"
+config_filename = "Starling-Caption-Credit-config.json"
+
+# Load config
+with open(config_filename, "r") as config_file:
+    config = json.load(config_file)
+
+with open(filename, newline="\n", encoding="utf8") as csvfile:
+    csv_reader = csv.reader(
+        csvfile,
+        delimiter=",",
+    )
+    heading = None
+
+    countlines = 0
+    json_metadata_template = {}
+
+    result = []
+    for row in csv_reader:
+        # Read Heading
+        if heading == None:
+            column_index = 0
+            for col_name in row:
+                if col_name == "":
+                    col_name = "col_" + str(column_index)
+                json_metadata_template[col_name] = ""
+                column_index += 1
+            heading = row
+        else:
+
+            countlines = countlines + 1
+
+            json_metadata = deepcopy(json_metadata_template)
+            column_index = 0
+            for item in row:
+                if len(item) > 1 and item[1] == "[":
+                    json_metadata[heading[column_index]] = ast.literal_eval(item)
```

Comment:
Maybe wrap this in a try-except block so you still ingest values even if they can't be evaluated. Like `[foo, bar]`.

Reply:
```suggestion
                    try:
                        json_metadata[heading[column_index]] = ast.literal_eval(item)
                    except:
                        json_metadata[heading[column_index]] = item
```